integration/jnifuse/jnifuse-java/pom.xml

```diff
@@ -0,0 +1,54 @@
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>jnifuse</artifactId>
+    <groupId>org.alluxio</groupId>
+    <version>2.5.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>jnifuse-java</artifactId>
+
+  <properties>
+    <!-- The following paths need to be defined here as well as in the parent pom so that mvn can -->
+    <!-- run properly from sub-project directories -->
+    <build.path>${project.parent.parent.parent.basedir}/build</build.path>
+    <jnifuse-native.version>${project.version}</jnifuse-native.version>
```

Comment:
why do we need different version?

Reply:
Maybe we can use jnifuse-native from maven repository, but build jnifuse-java in the alluxio project.